include/sway/criteria.h

```diff
@@ -14,29 +14,33 @@ enum criteria_type {
 	CT_NO_FOCUS                = 1 << 4,
 };
 
+struct pattern {
+	char *literal;
```

Comment:
Can we instead make this an enum?

```c
enum pattern_type {
    PATTERN_PCRE,
    PATTERN_FOCUSED,
    // etc
}
```

Reply:
Yeah that does sound better. I was trying a little too hard to match the i3 code.

I've switched it to an enum, PTAL.